3-enrich/emr-etl-runner/lib/snowplow-emr-etl-runner/emr_job.rb

```diff
@@ -677,8 +737,8 @@ def get_rdb_loader_steps(config, targets, resolver, jar, rdbloader_steps)
             "--resolver", default_arguments[:resolver],
             "--logkey", log_key,
             "--target", encoded_target
-          ] + unless rdbloader_steps[:skip].empty?
-            ["--skip", rdbloader_steps[:skip].join(",")]
+          ] + unless skip_steps.empty?
```

Comment:
Here and L744:
Consider changing the ` + unless ` to use a variable or the ternary operator to make more readable.